### PR TITLE
Cif 315 improve consistency and standardization of cif code for layers

### PR DESCRIPTION
**Pull request by kcartier-wri**
Restoring import statements that the previous merge had removed since I saw changed behavior in thermal-comfort-modeling after removing the imports. I verified that restoring them brings the behavior back to the previous state.

The "from osgeo import gdal" statement in layers.py seemed to be the specific import that changed the behavior that I saw.

Many of these imports appear to be unused and should be further investigated for potential removal at a later date.

### CODE REVIEW

File: city_metrix/layers/urban_land_use.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
 import ee
 
 from .layer import Layer, get_image_collection
```
Comment (chrowe) [inline city_metrix/layers/urban_land_use.py:6]: Should this be `from .layer import Layer, get_utm_zone_epsg, get_image_collection`?

File: city_metrix/layers/world_pop.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
 import ee
 
 from .layer import Layer, get_image_collection
```
Comment (chrowe) [inline city_metrix/layers/world_pop.py:6]: Should this be `from .layer import Layer, get_utm_zone_epsg, get_image_collection`?

File: city_metrix/layers/tree_cover.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+from .layer import Layer, get_utm_zone_epsg, get_image_collection
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
 import ee
 
 from .layer import Layer, get_image_collection
```
Comment (chrowe) [inline city_metrix/layers/tree_cover.py:7]: Do you want this or what is in line 1?

File: city_metrix/layers/tree_canopy_height.py [hunk 4]
```diff
@@ -1,3 +1,7 @@
+from .layer import Layer, get_utm_zone_epsg, get_image_collection
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
 import ee
 
 from .layer import Layer, get_image_collection
```
Comment (chrowe) [inline city_metrix/layers/tree_canopy_height.py:7]: Do you want this or what is in line 1?

**Review by chrowe — COMMENTED** [review]
Based on https://github.com/wri/cities-cif/compare/c270fd007985316bd9f9f1f76ea0e9bd2be87a4a...CIF-315-Improve-consistency-and-standardization-of-CIF-code-for-Layers I had a few import questions.

File: city_metrix/layers/urban_land_use.py (hunk 1 above)
Comment (kcartier-wri) [reply city_metrix/layers/urban_land_use.py]: fixed

File: city_metrix/layers/world_pop.py (hunk 2 above)
Comment (kcartier-wri) [reply city_metrix/layers/world_pop.py]: fixed

File: city_metrix/layers/tree_cover.py (hunk 3 above)
Comment (kcartier-wri) [reply city_metrix/layers/tree_cover.py]: fixed

File: city_metrix/layers/tree_canopy_height.py (hunk 4 above)
Comment (kcartier-wri) [reply city_metrix/layers/tree_canopy_height.py]: fixed

**Review by chrowe — APPROVED** [review]

**Pull request merged**